client/app/main/student/student.controller.js

```diff
@@ -13,6 +14,11 @@ function StudentCtrl($scope, $state, $stateParams, Student, toastr) {
     });
   });
 
+  AbsenceRecord.student({id: $stateParams.id})
+    .$promise.then(function(total) {
+      $scope.percentage = Math.floor((total.present / total.enrolled) * 100);
+    });
+
```

Comment:
I'm thinking that this shouldn't be a separate API call. We could pull and calculate this data along with other data we want to display in the `Student.get` api call.